### PR TITLE
feat: Adding new Trust Icons

### PR DESCRIPTION
**Pull request by alotoftype**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Related Tickets & Documents

Resolves #938 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Screen Shot 2021-04-21 at 18 30 28

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
No

## [optional] What gif best describes this PR or how it makes you feel?
Nervous nelly

### CODE REVIEW

**Comment by alotoftype** [comment]
@bdougie Thanks for the assist. Missed out on the feature name title this is a good assortment of tests that you are running. how do I find more information about running PR audits. this is pretty awesome.

**Comment by bdougie** [comment]
> @bdougie Thanks for the assist. Missed out on the feature name title this is a good assortment of tests that you are running. how do I find more information about running PR audits. this is pretty awesome. 

Those are mostly GitHub Actions. If you want a quick intro I have a "what are GitHub actions" video in this playlist https://youtube.com/playlist?list=PLpURC3VhaQD29vdGBngvrIapNRjLIq64x

Can you also link the issue you are working on in the pull request description? 

https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

**Comment by alotoftype** [comment]
@bdougie linked the issue is now included

File: src/components/Hero.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import {Link} from "react-router-dom";
 import {MarketingButton} from "../styles/Button";
 import {Wrapper, PizzaPosition, TopHero, SubHero, TrustSection, QuoteSection, DetailsSection} from "../styles/Header";
-import {gucci, go, graphql, electron, next, sauceFull, bdougie} from "../images";
```
Comment (bdougie) [inline src/components/Hero.js:5]: @alotoftype can you also delete these files from the repo?

**Review by bdougie — CHANGES_REQUESTED** [review]
This looks good to go, just need to delete the images we are not using anymore

File: src/components/Hero.js (hunk 1 above)
Comment (alotoftype) [reply src/components/Hero.js:5]: yup will do

File: src/images/index.js [hunk 2]
```diff
@@ -8,7 +8,12 @@ import go from "./go.png";
 import electron from "./electron.png";
 import next from "./next.png";
 import gucci from "./gucci.jpg";
+import gitTrends from "./gitTrends.svg";
 import graphql from "./graphql.png";
```
Comment (filiptronicek) [inline src/images/index.js:12]: Seems like the images exports got broken, this should fix that
```suggestion
import gucci from "./gucci.jpg";
import gitTrends from "./gitTrends.svg";
```

File: src/images/index.js [hunk 3]
```diff
@@ -8,7 +8,12 @@ import go from "./go.png";
...
 import ohno from "./404.svg";
+import remirror from "./remirror.svg";
+import UseShoppingCart from "./UseShoppingCart.png";
 
-export {gucci, go, graphql, electron, ohno, next, bdougie, face, sauce, sauced, pizza, sauceFull};
+export {gucci, go, graphql, electron, ohno, next, bdougie, face, sauce, sauced, pizza, sauceFull, remirror,
```
Comment (filiptronicek) [inline src/images/index.js:17]:
```suggestion
export {gucci, ohno, bdougie, face, sauce, sauced, pizza, sauceFull, remirror,
```

File: src/images/index.js [hunk 4]
```diff
@@ -5,10 +5,12 @@ import sauced from "./sauced.svg";
...
-import next from "./next.png";
 import gucci from "./gucci.jpg";
-import graphql from "./graphql.png";
+import gitTrends from "./gitTrends.svg";
 import ohno from "./404.svg";
+import remirror from "./remirror.svg";
+import UseShoppingCart from "./UseShoppingCart.png";
 
-export {gucci, go, graphql, electron, ohno, next, bdougie, face, sauce, sauced, pizza, sauceFull};
+export {gucci, graphql, ohno, bdougie, face, sauce, sauced, pizza, sauceFull, remirror,
```
Comment (filiptronicek) [inline src/images/index.js:14]:
```suggestion
export {gucci, ohno, bdougie, face, sauce, sauced, pizza, sauceFull, remirror,
```

File: src/images/index.js [hunk 5]
```diff
@@ -5,10 +5,12 @@ import sauced from "./sauced.svg";
 import sauceFull from "./pizza-n-sauce.svg";
 import pizza from "./pizza.svg";
 import go from "./go.png";
```
Comment (filiptronicek) [inline src/images/index.js:7]:
```suggestion
```

File: src/images/index.js [hunk 6]
```diff
@@ -4,11 +4,12 @@ import sauce from "./sauce.svg";
 import sauced from "./sauced.svg";
 import sauceFull from "./pizza-n-sauce.svg";
 import pizza from "./pizza.svg";
-import go from "./go.png";
-import electron from "./electron.png";
-import next from "./next.png";
 import gucci from "./gucci.jpg";
```
Comment (filiptronicek) [inline src/images/index.js:7]:
```suggestion
import gucci from "./gucci.jpg";
import graphql from "./graphql.png";
```

File: src/images/index.js [hunk 7]
```diff
@@ -4,11 +4,12 @@ import sauce from "./sauce.svg";
...
-import graphql from "./graphql.png";
+import gitTrends from "./gitTrends.svg";
 import ohno from "./404.svg";
+import remirror from "./remirror.svg";
+import UseShoppingCart from "./UseShoppingCart.png";
 
-export {gucci, go, graphql, electron, ohno, next, bdougie, face, sauce, sauced, pizza, sauceFull};
+export {gucci, ohno, bdougie, face, sauce, sauced, pizza, sauceFull, remirror,
```
Comment (filiptronicek) [inline src/images/index.js:13]:
```suggestion
export {gucci, ohno, bdougie, face, graphql, sauce, sauced, pizza, sauceFull, remirror,
```

**Review by bdougie — APPROVED** [review]
LGTM!

**Pull request merged**